man/iocost.conf.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
+
+<refentry id="iocost.conf" xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>iocost.conf</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>iocost.conf</refentrytitle>
+    <manvolnum>5</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>iocost.conf</refname>
+    <refname>iocost.conf.d</refname>
+    <refpurpose>Configuration file for the iocost solution manager</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <para><filename>/etc/systemd/iocost.conf</filename></para>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para>This file configures the behavior of <literal>iocost</literal>, a tool mostly used by udev
+    rules to automatically apply I/O cost solutions to <literal>/sys/fs/cgroup/io.cost.*</literal>.
+
+    The qos and model values are calculated based on benchmarks collected on the
+    <literal>iocost-benchmark</literal> github project and turned into a set of solutions
+    that go from most to least isolated. Isolation allows the system to continue responsive
+    at the cost of bandwidth.
+
+    Which solutions are available for a device can be queried from the udev metadata attached to it. By
+    default the naive solution is used, which provides the most bandwidth.</para>
+  </refsect1>
+
+  <xi:include href="standard-conf.xml" xpointer="main-conf" />
+
+  <refsect1>
+    <title>Options</title>
+
+    <para>All options are configured in the [IOCost] section:</para>
+
+    <variablelist class='config-directives'>
+
+      <varlistentry>
+        <term><varname>TargetSolution=</varname></term>
+
+        <listitem><para>Chooses which I/O cost solution to use for the devices in this system.
+        The known solutions can be queried from the udev metadata attached to the devices. If
+        a device does not have the specified solution, the first one listed in
+        <literal>IOCOST_SOLUTIONS</literal> is used instead.</para></listitem>
```

Comment:
this should say what a "solution" is identified with here, i.e. say "an identifying name string" or something, ad maybe give an example